020-Getting-started/080-astro.mdx

```diff
@@ -50,153 +51,713 @@ apiKey=YOUR_API_KEY_HERE
 
 ## Create a new Astro app
 
-To start lets create a new Astro project. Set it up with typescript support so that we can utilize the SDK later in this tutorial. 
+Let's begin by creating a new Astro application that makes use of a template setup to use Tailwind CSS for some styling.
+Run the following command and accept all the prompt defaults:
+
+```sh
+npm create astro@latest -- --template with-tailwindcss xata-astro
```

Comment:
Didn't notice any install instructions for tailwind with the Next JS version of this guide. Are we missing a step there? I don't think it comes by default?